roles/openshift_service_catalog/templates/api_server.j2

```diff
@@ -0,0 +1,86 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    app: apiserver
+  name: apiserver
+spec:
+  selector:
+    matchLabels:
+      app: apiserver
+  updateStrategy:
+    rollingUpdate:
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: apiserver
+    spec:
+      serviceAccountName: service-catalog-apiserver
+      nodeSelector:
+{% for key, value in node_selector.iteritems() %}
+          {{key}}: "{{value}}"
+{% endfor %}
+      containers:
+      - args:
+        - --storage-type
+        - etcd
+        - --secure-port
+        - "6443"
+        - --insecure-bind-address
+        - 0.0.0.0
+        - --insecure-port
+        - "8081"
+        - --etcd-servers
+        - http://localhost:2379
```

Comment:
this will need to point to the core's etcd instead. TODO: remove embedded etcd container too